config/game_options.txt

```diff
@@ -481,3 +481,8 @@ MAXFINE 2000
 ## Comment if you wish to enable title music playing at the lobby screen. This flag is disabled by default to facilitate better code testing on local machines.
 ## Do keep in mind that this flag will not affect individual player's preferences: if they opt-out on your server, it will never play for them.
 DISALLOW_TITLE_MUSIC
+
+## Max distance for clients to view with widescren mode enabled. 
+MAX_CLIENT_VIEW_X 19
+MAX_CLIENT_VIEW_Y 16
+
```

Comment:
please don't make the defaults larger then the current max

Reply:
roger